e2e/tests/components/date-time-picker/date-time-picker.e2e-spec.ts

```diff
@@ -200,4 +200,22 @@ describe('Date Time Picker Tests', () => {
         }
 
     });
+
+    it('should correctly set the date when select an adjacent month', async () => {
+
+        const date = await page.getDate('31');
+        await date.click();
+
+        // check if it now has the active class
+        const classes = await date.getAttribute('class');
```

Comment:
You need to move this too.

Reply:
Yes, the markup has changed so you have to fetch the new button element and test it.